guides/7-jlp/4-JLP-Economics.md

```diff
@@ -0,0 +1,142 @@
+---
+sidebar_label: "JLP Economics"
+description: "JLP Economics"
+title: "JLP Economics"
+---
+
+
+
+## The JLP Token
+
+The [JLP token](https://birdeye.so/token/27G8MtK7VtTcCHkpASjSDdkWWYfoqT6ggEuKidVJidD4?chain=solana) is the token given to users who provide liquidity to the JLP pool. JLP can be acquired either through [Jupiter Swap](https://jup.ag/) or through the [Earn page](https://jup.ag/perps-earn), which mints or burns JLP tokens directly from the pool.
+
+More information on how to get JLP [here](./3-How-To-Get-JLP.md).
+
+## Custodies
+
+The JLP pool manages several custodies (tokens) for liquidity providers:
+
+* SOL: [7xS2gz2bTp3fwCC7knJvUWTEU9Tycczu6VhJYKgi1wdz](https://solscan.io/account/7xS2gz2bTp3fwCC7knJvUWTEU9Tycczu6VhJYKgi1wdz)
+* ETH: [AQCGyheWPLeo6Qp9WpYS9m3Qj479t7R636N9ey1rEjEn](https://solscan.io/account/AQCGyheWPLeo6Qp9WpYS9m3Qj479t7R636N9ey1rEjEn)
+* BTC: [5Pv3gM9JrFFH883SWAhvJC9RPYmo8UNxuFtv5bMMALkm](https://solscan.io/account/5Pv3gM9JrFFH883SWAhvJC9RPYmo8UNxuFtv5bMMALkm)
+* USDC: [G18jKKXQwBbrHeiK3C9MRXhkHsLHf7XgCSisykV46EZa](https://solscan.io/account/G18jKKXQwBbrHeiK3C9MRXhkHsLHf7XgCSisykV46EZa)
+* USDT: [4vkNeXiYEUizLdrpdPS1eC2mccyM4NUPRtERrk6ZETkk](https://solscan.io/account/4vkNeXiYEUizLdrpdPS1eC2mccyM4NUPRtERrk6ZETkk)
+
+More info on the `Custody` account is explained in [Onchain Account Types](../8-perpetual-exchange/3-onchain-accounts.md).
+
+### Assets Under Management
+
+The AUM for each `Custody` account in the pool is calculated as follows:
+
+#### Stablecoins (USDC, USDT)
+
+`aum = owned * current_price`
+
+#### Non-stablecoins (SOL, ETH, wBTC)
+
+1. Calculate the global short position profit/loss (Unrealized PnL in USD):
+
+`unrealized_pnl = (global_short_sizes * (|global_short_average_prices - current_price|)) / global_short_average_prices)`
+
+:::tip
+If `current_price` > `global_short_average_prices`, traders are losing on short positions.
+:::
+
+2. Net Asset Value (NAV):
+
+`available_tokens = owned - locked`
+
+`nav = available_tokens * current_token_price`
+
+`nav += guaranteed_usd`
+
+:::info
+The `guaranteed_usd` value in each `Custody` account represents an estimate of the total size of all long positions. It is only an estimate as `guaranteed_usd` is only updated when positions are updated (i.e. opening / closing positions, updating collateral). It does not update in real-time when asset prices change. 
```

Comment:
guaranteed_usd = size of all trades in USD value - collateral of all trades in USD value.